save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt

```diff
@@ -194,7 +194,7 @@ fun topBar() = fc<TopBarProps> { props ->
                     attrs["aria-labelledby"] = "userDropdown"
                     dropdownEntry(faCog, "Settings") {
                         attrs.onClickFunction = {
-                            window.location.href = "#/${props.userInfo?.name}/settings/email"
+                            window.location.href = "#/${props.userInfo?.name ?: "Unauthenticated"}/settings/email"
```

Comment:
Maybe, we can already disable it if `props.userInfo?.name == null`? What's the point of showing settings view to anonymous users?

Reply:
Agree. Fixed